rs/sns/governance/src/types.rs

```diff
@@ -2750,6 +2750,24 @@ pub mod test_helpers {
     }
 }
 
+impl From<upgrade_journal_entry::UpgradeStepsRefreshed> for upgrade_journal_entry::Event {
+    fn from(event: upgrade_journal_entry::UpgradeStepsRefreshed) -> Self {
+        upgrade_journal_entry::Event::UpgradeStepsRefreshed(event)
+    }
+}
+impl From<upgrade_journal_entry::UpgradeStarted> for upgrade_journal_entry::Event {
+    fn from(event: upgrade_journal_entry::UpgradeStarted) -> Self {
+        upgrade_journal_entry::Event::UpgradeStarted(event)
+    }
+}
+impl From<upgrade_journal_entry::UpgradeOutcome> for upgrade_journal_entry::Event {
+    fn from(event: upgrade_journal_entry::UpgradeOutcome) -> Self {
+        upgrade_journal_entry::Event::UpgradeOutcome(event)
+    }
+}
+// Note, we do not implement From<upgrade_journal_entry::TargetVersionSet> for upgrade_journal_entry::Event
```

Comment:
Let's split the api types `TargetVersionSet` and `TargetVersionReset`, after all, to avoid this special case.

Reply:
Done!